webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -31,7 +48,21 @@ const CardItem = ({ content, header, children }) => {
   return (
     <GridItem xl2={3} md={6} lg={5}>
       <Card isHoverable>
-        <CardTitle>{header}</CardTitle>
+        {dropdownItems && (
```

Comment:
Should this condition only be for the `CardActions` component? otherwise the `CardTitle` won't show if there's no dropdown